addon/components/ui-calendar/template.hbs

```diff
@@ -1,54 +1,55 @@
-<nav local-class="nav">
-  <button 
-    type="button" 
-    local-class="nav-control on-previous-month" 
+<nav local-class="nav" class="ui-calendar-nav">
+  <button
+    type="button"
+    local-class="nav-control on-previous-month" class="ui-calendar-nav-previous"
     {{on "click" (fn this.onMoveCenter -1 "month")}}
   >
-    {{ui-icon "arrow-left-12" size="small"}}
+    {{svg-jar "arrow-left-1" height="15" width="15"}}
   </button>
-  <div local-class="nav-title">
+  <div local-class="nav-title" class="ui-calendar-nav-title">
     {{moment-format this.center "MMMM YYYY"}}
   </div>
-  <button 
+  <button
     type="button"
-    local-class="nav-control on-next-month" 
+    local-class="nav-control on-next-month" class="ui-calendar-nav-next"
     {{on "click" (fn this.onMoveCenter 1 "month")}}
   >
-    {{ui-icon "arrow-right-12" size="small"}}
+    {{svg-jar "arrow-right-1" height="15" width="15"}}
```

Comment:
The icon used has been changed from `arrow-right-12` to `arrow-right-1` here - is that right?

Reply:
There are two things here, there was a reference to ui-icon which didn't exist, and arrow-right-12 which also didn't exist. So I changed it to use svg-jar and arrow-right-1

Reply:
See other issue